### ISSUE TITLE
Scratch Org creation fails when creating from Org Shape

### ISSUE THREAD

**Issue by pranitjain1** [forcedotcom/cli#1758]
### Summary

When trying to create Scratch org using sfdx force:org:create command using org shape, it is failing to create scratch org.

### Steps To Reproduce:

1. Create a project-scratch-def file using soruceOrg (org shape)
2. Run sfdx force:org:create -f config/project-scratch-def.json -w 30 command
3. It fails to create Scratch org with below error
A fatal signup error occurred. Please try again. If you still see this error, contact Salesforce Support for assistance 

Try this: 
See https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_signuprequest.htm for information on error code C-9999.

### Expected result

Scratch Org gets created using Org Shape

### Actual result

Failing with Error:
A fatal signup error occurred. Please try again. If you still see this error, contact Salesforce Support for assistance 

Try this: 
See https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_signuprequest.htm for information on error code C-9999.

### System Information

- Which shell/terminal are you using? (e.g. bash, zsh, powershell 5, powershell 7, cmd.exe, etc.)

{
  "cliVersion": "sfdx-cli/7.172.0",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v16.17.1",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.3.1 (core)",
    "@oclif/plugin-commands 2.2.0 (core)",
    "@oclif/plugin-help 5.1.14 (core)",
    "@oclif/plugin-not-found 2.3.3 (core)",
    "@oclif/plugin-plugins 2.1.1 (core)",
    "@oclif/plugin-update 3.0.2 (core)",
    "@oclif/plugin-version 1.1.2 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.6 (core)",
    "@oclif/plugin-which 2.1.0 (core)",
    "alias 2.1.5 (core)",
    "apex 1.2.0 (core)",
    "auth 2.2.13 (core)",
    "community 2.0.3 (core)",
    "config 1.4.21 (core)",
    "custom-metadata 2.0.1 (core)",
    "data 2.1.4 (core)",
    "generator 2.0.5 (core)",
    "info 2.1.9 (core)",
    "limits 2.0.5 (core)",
    "org 2.2.7 (core)",
    "packaging 1.9.4 (core)",
    "schema 2.1.10 (core)",
    "signups 1.2.7 (core)",
    "source 2.0.17 (core)",
    "telemetry 2.0.2 (core)",
    "templates 55.1.0 (core)",
    "trust 2.0.4 (core)",
    "user 2.1.10 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "salesforce-alm 54.8.1 (core)"
  ],
  "osVersion": "Darwin 21.6.0",
  "shell": "zsh",
  "rootPath": "/Users/pranit.jain/.local/share/sfdx/client/7.172.0-1aa6bdf"
}
### Additional information

Log file attached.
[SignUp.log](https://github.com/forcedotcom/cli/files/9798529/SignUp.log)

**Comment by AndrewRayCode** [comment]
I'm having similar issues creating scratch orgs.  I created a stackoverflow question for it: https://salesforce.stackexchange.com/questions/387551/sfdx-scratch-org-creation-extremely-unstable-unusable

Note that Salesforce has had a months long massive scratch org outage: https://github.com/forcedotcom/cli/issues/1690

Given Sunday, October 16th's global Salesforce outage, and how the incident status was marked as resolved long before the incident was resolved, we know that incident updates aren't trustworthy from Salesforce. I suspect there are still major ongoing issues with scratch orgs.

I have received your signup error, and the most recent error I'm getting is:

> ERROR running force:org:create:  The client has timed out.

It currently takes around 5 retries for me to get a scratch org created using `sfdx`.
